File: AppSettingsManagerAPI/AppSettingsManagerApi.Model/Requests/PermissionRequestResponse.cs
namespace AppSettingsManagerApi.Model.Requests;

public class PermissionRequestResponse
{
    public string UserId { get; set; }
    public string SettingGroupId { get; set; }
    public bool Approved { get; set; }
    public string ApproverId { get; set; }
    public string Password { get; set; }
}

File: AppSettingsManagerUI/src/app/services/bff/models/PermissionRequestResponse.ts
export class PermissionRequestResponse {
    UserId: string;
    SettingGroupId: string;
    Approved: boolean;
    ApproverId: string;
    Password: string;

    constructor(userId: string, settingGroupId: string, approved: boolean, approverId: string, password: string){
        this.UserId = userId;
        this.SettingGroupId = settingGroupId;
        this.Approved = approved;
        this.ApproverId = approverId;
        this.Password = password;
    }
}